yellowbrick/features/importances.py

```diff
@@ -86,7 +86,7 @@ class FeatureImportances(ModelVisualizer):
     >>> visualizer.poof()
     """
 
-    def __init__(self, model, ax=None, labels=None, relative=True,
+    def __init__(self, model, ax=None, labels=None, relative=True, color_neg='b', color_pos='g',
```

Comment:
One minor change, can you please update the documentation for this visualizer class? It should be a pretty straightforward change located in this section of the code:

https://github.com/kim81682/yellowbrick/blob/3489dd1a7b97901919e1bc2fcbafbaa8a5a20cc9/yellowbrick/features/importances.py#L55-L70

Again, thanks for contributing this!